app/v2/models/HeaderType.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.models
+
+import play.api.http.HeaderNames
+import play.api.http.MimeTypes
+
+sealed abstract class HeaderType(val header: (String, String)*)
+
+object HeaderTypes {
+
+  case object jsonToXml
+      extends HeaderType(
+        (HeaderNames.CONTENT_TYPE -> MimeTypes.JSON),
+        (HeaderNames.ACCEPT       -> MimeTypes.XML)
+      )
+
+  case object xmlToJson
```

Comment:
I may be wrong, but I don't think the XML -> JSON case is required here since it will be the auditing service that will send the request to convert the XML to JSON?

Reply:
Not sure - probably need to Tim or Dan to clarify!

Reply:
We might need it for the Web FE, they _might_ want the messages converted to Json if they poll our service. If so, we'll probably need to do it here.

@crystalmark might have a bit of a better idea as to how we're going to work with the web front end on this regard -- indeed, we may need to talk to them about it.